fabfile.py

```diff
@@ -1508,6 +1641,8 @@ def commonenv(projectpath, venvpath=None):
     # It is recommended you keep localhost even if you have access to
     # unix domain sockets, it's more portable across different pg_hba configurations.
     env.db_host = 'localhost'
+    if config.has_option("app:assembl", "db_host"):
```

Comment:
very different behaviour if talking to postgres through a tcp socket (other host) or pg_ctl (local)